lib/src/merged_tree.rs

```diff
@@ -607,7 +607,6 @@ pub struct TreeDiffIterator<'matcher> {
     store: Arc<Store>,
     stack: Vec<TreeDiffItem>,
     matcher: &'matcher dyn Matcher,
-    copy_records: &'matcher CopyRecords,
```

Comment:
This field was added in one commit, then removed in the next, and then added back again. So keep it in the middle commit, i guess.

Reply:
removed from this comment, so it isn't here until it is used